src/seedu/addressbook/ui/TextUi.java

```diff
@@ -87,7 +87,7 @@ public String getUserCommand() {
         showToUser("[Command entered:" + fullInputLine + "]");
         return fullInputLine;
     }
-
+   
```

Comment:
Avoid formatting unrelated parts of the code. Doing so can result in unrelated changes being included in the commit. You can use sourcetree to look through changes before committing, and discard any unrelated changes.